frontend/e2e-tests/dom-to-db-tests/data-entry.e2e.ts

```diff
@@ -15,7 +15,80 @@ import {
 } from "e2e-tests/page-objects/data_entry";
 
 import { test } from "./fixtures";
-import { pollingStation33 } from "./test-data/PollingStationTestData";
+import {
+  noErrorsWarningsResponse,
+  noRecountNoDifferencesRequest,
+  pollingStation33,
+} from "./test-data/PollingStationTestData";
+
+async function fillDataEntryNoRecountNoDifferences(page: Page) {
```

Comment:
This needs refactoring in a new issue. These changes have given me a clearer picture of where we're outgrowing our current d2d test framework.

Some of the things I'd like to (see) tackle(d) in that refactor:
- This function (and thus the test this code came from) does too many things. Do we need it to check focus and if the value shows up in the field? And even if so, perhaps we can split the focus etc test from the tests that cover filling first/second data entry.
- It would be useful to be able to use the same constant for both API (sending request or checking request) and UI (filling in the data entry forms).
- `PollingStationTestData.ts` contains fixture data, a request template, a request, and an expected error response. These are different kinds of things, so should be split out.
- Instead of hard-coding the values of `pollingStation33` in a test data file, us an API call in a fixture to retrieve the data from the backend.
- Making the code consistent in how it checks it's on the right page (replacing `waitFor()`s with `expect`s on `toBeVisible()`.
- Moving the first data entry of the test "equal second data entry after first data entry" into a fixture.

Reply:
https://github.com/kiesraad/abacus/issues/665